pinot-common/src/main/java/org/apache/pinot/common/utils/fetcher/SegmentFetcherFactory.java

```diff
@@ -182,6 +183,12 @@ public static void fetchAndDecryptSegmentToLocal(String uri, File dest, String c
     getInstance().fetchAndDecryptSegmentToLocalInternal(uri, dest, crypterName);
   }
 
+  // uris have equal status
```

Comment:
The comment lacks details: do you mean each uri in the list has equal prob being selected for segment download?